bitnami/nginx/templates/deployment.yaml

```diff
@@ -35,6 +35,17 @@ spec:
       {{- include "nginx.imagePullSecrets" . | nindent 6 }}
       {{- if .Values.affinity }}
       affinity: {{- include "common.tplvalues.render" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        {{- if not (empty .Values.podAffinityPreset) }}
+        podAffinity: {{- include "common.affinities.pod" (dict "type" .Values.podAffinityPreset "context" $) | nindent 10 }}
+        {{- end }}
+        {{- if not (empty .Values.podAntiAffinityPreset) }}
+        podAntiAffinity: {{- include "common.affinities.pod" (dict "type" .Values.podAntiAffinityPreset "context" $) | nindent 10 }}
+        {{- end }}
+        {{- if not (empty .Values.nodeAffinityPreset) }}
+        nodeAffinity: {{- include "common.affinities.node" (dict "type" .Values.nodeAffinityPreset "key" "some-label-key" "values" (list "some-label-value")) | nindent 10 }}
```

Comment:
I imagine `some-label-key` should not be there, right?

Reply:
+1